### PR TITLE
[T2A2][F11] ZhuangYufeng

### PR DESCRIPTION
**Pull request by rainy-windy**
Ready for Review

### CODE REVIEW

**Comment by jeffryhartanto** [comment]
Please mention your team ID, e.g. [F11-A1]. Make sure you use the correct title next time.

**Comment by cmkumar87** [comment]
Please fix your PR title as [T2A2][F11-A4] and do so int he future. Had a hard time tracking your PR :(

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -44,7 +45,7 @@
     /**
      * A decorative prefix added to the beginning of lines printed by AddressBook
      */
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:47]: Good! Variables should generally be local to a class. 
However, constants that are of used across the application (across many classes) should be consolidated in a single class as static final variables with public access.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -202,13 +196,20 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        interpretInputAndExecuteTillExit();
+    }
+
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:201]: Ensure indentations are as per coding style manual after code changes. eclipse auto format makes this easy to do!

**Pull request closed without merging**